compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/test/distribute_inner_tiled_to_lanes.mlir

```diff
@@ -571,6 +571,148 @@ func.func @data_tiled_2x2x4_tensor_multi_mma_unrolled_to_subgroups(%lhs: tensor<
 
 // -----
 
+#scaled_contraction_accesses = [
+  affine_map<(m, n, k, kb) -> (m, k, kb)>,
+  affine_map<(m, n, k, kb) -> (n, k, kb)>,
+  affine_map<(m, n, k, kb) -> (m, k)>,
+  affine_map<(m, n, k, kb) -> (n, k)>,
+  affine_map<(m, n, k, kb) -> (m, n)>
+]
+func.func @data_tiled_scaled_2x2x4_tensor_multi_mma_unrolled_to_subgroups(
```

Comment:
For my understanding, how is this 'unrolled'? I don't see that in the checks? Or does this refer to it being unrolled when lowered to mfmas?